src/analysis/inner_product_space/basic.lean

```diff
@@ -1199,7 +1203,8 @@ variables {E'' : Type*} [inner_product_space 𝕜 E'']
 
 /-- A linear isometry preserves the inner product. -/
 @[simp] lemma linear_isometry.inner_map_map (f : E →ₗᵢ[𝕜] E') (x y : E) : ⟪f x, f y⟫ = ⟪x, y⟫ :=
-by simp [inner_eq_sum_norm_sq_div_four, ← f.norm_map]
+by simp_rw [inner_eq_sum_norm_sq_div_four, ← f.norm_map, linear_isometry.map_add,
+            linear_isometry.map_sub, linear_isometry.map_smul]
```

Comment:
@hrmacbeth: I'm generally in favor of changes from `simp` to `simp_rw` like this one, as the resulting proof is less fragile and possible to navigate in the infoview.

Reply:
This seems more like a style choice.  I do not think this opinion is widely enough shared to make it acceptable to introduce it to other people's code.  You wouldn't change docstrings someone else wrote to adjust from English to American spelling.

Reply:
not really, it's more like adding an extra explanation to someone's docstring

Reply:
On the contrary, to me these changes read as obfuscation, making it harder to identify which parts of the proof have content (hint: the parts that couldn't be `simp`).  Anyway, there's no point getting into heated debate on a PR that three people are reading.  If you really want to change the style guide of mathlib away from plain `simp`s, please start a discussion on Zulip #general.